Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -43,12 +45,18 @@
     help="Use the Ml Ambiguity Solver instead of the classical one",
     action="store_true",
 )
+parser.add_argument(
+    "--MLSeedFilter",
+    help="Use the Ml seed filter to select seed after the seeding step",
+    action="store_true",
+)
```

Comment:
Cue my obligatory comment that I'm not a fan of just adding extra features to this script steered by command line options.

Reply:
I don't disagree and I am 100% ready to implement this any other way (but I am not sure which one)